gpu/kinfu/src/cuda/ray_caster.cu

```diff
@@ -273,7 +273,7 @@ namespace pcl
             break;
           }
 
-        }          /* for(;;)  */
+        }          /* for(;)  */
```

Comment:
That's not how `;;` works, but I think it'd be better to remove the comment here

Reply:
I was trying to get rid of double '::', I think I fat-fingered 'Replace' in this instance